pom.xml

```diff
@@ -149,6 +154,12 @@
             <version>3.0</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>xalan</groupId>
+            <artifactId>xalan</artifactId>
+            <version>2.7.2</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
This dependency is the reason for a test failure:
[ERROR] Tests run: 15, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 227.839 s <<< FAILURE! - in hudson.plugins.jobConfigHistory.JobConfigBadgeActionTest
[ERROR] testOnAttached(hudson.plugins.jobConfigHistory.JobConfigBadgeActionTest)  Time elapsed: 180.033 s  <<< ERROR!
org.junit.runners.model.TestTimedOutException: test timed out after 180 seconds
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:502)
	at org.jvnet.hudson.reactor.Reactor.execute(Reactor.java:280)
	at jenkins.InitReactorRunner.run(InitReactorRunner.java:48)
	at jenkins.model.Jenkins.executeReactor(Jenkins.java:1117)
	at jenkins.model.Jenkins.<init>(Jenkins.java:921)
	at hudson.model.Hudson.<init>(Hudson.java:85)
	at org.jvnet.hudson.test.JenkinsRule.newHudson(JenkinsRule.java:674)
	at org.jvnet.hudson.test.JenkinsRule.before(JenkinsRule.java:401)
	at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:594)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

Removing the scope has no effect.
Which java version did you used? I'm using JDK 1.8.

Any suggestions?